### ISSUE TITLE
Enhance syntax support for attribute arrays

### ISSUE THREAD

**Issue by timurproko** [melMass/vscode-vex#9]
Hi, I have applied custom colors to match the syntax highlight in Visual Studio Code with the Houdini editor.
I have noticed that Visual Studio syntax doesn't recognize the attribute arrays likle - s[]@names.
Visual studio doesn't have textmate scope for such items so the color highlight doesn't match.
Could be nice to add this item to syntax to have 1-to-1 matching.
![Houdini](https://user-images.githubusercontent.com/80062439/178144051-96c65d0d-4cee-41dc-84dd-9028f1068157.png)
![VisualStudio](https://user-images.githubusercontent.com/80062439/178144056-5cda154a-b7eb-4f4e-882d-29aa392ac737.png)

**Comment by melMass** [comment]
Hi Tim,

Yes good point, I will release a `@next` version this month where I changed a few things I will look into your suggestion then, thanks for the feedback.

**Comment by timurproko** [comment]
Thanks for your reply. That’s will be awesome!

**Comment by timurproko** [comment]
Here is an additional case: the type of the variable and the function use the same textmate scope (storage.type.builtin.vex) and can't be colored differently as Houdini's native syntax highlight does.
![Snag_135dba67](https://user-images.githubusercontent.com/80062439/178263050-df7f17cc-6bf8-4189-845c-2e88081e5f89.png)
![Snag_135dd2a3](https://user-images.githubusercontent.com/80062439/178263066-74468d41-34c6-4b20-a110-599c541a93df.png)

Thanks again

**Comment by timurproko** [comment]
Also I am trying to get the function looks similar but can't get the same result
![image](https://user-images.githubusercontent.com/80062439/178307826-cdb55721-1ff3-4b9f-948a-8b3488fcee5b.png)
![image](https://user-images.githubusercontent.com/80062439/178307856-f2c3feb0-77ba-4a53-8764-f286df411a70.png)

**Comment by melMass** [comment]
@timurproko function is completely optional here. Use void if there is no return type.

Btw while I will look into these, the priority is not really text highlighting but more inline documentation/parameter descriptions

**Comment by timurproko** [comment]
I agree, but the color highlighting parity also helps when you switch back and forth between VS Code and Houdini Wrangle Node.
